### DISCUSSION TITLE
Snippets for Web Development (HTML, JavaScript, PHP, Vue)

### DISCUSSION THREAD

**Discussion by zhaolinlau** [geany/geany#3280] [Show and tell]
Hi guys, I've just made my version of snippets for my daily web development, and I also did a from A to Z's instruction on how to setup Geany for web development! Have a look at [Geany WebDev Snippets](https://github.com/zhaolinlau/Geany-WebDev-Snippets) <3

**Comment by elextr** [comment]
Perhaps you could put a link to it on the wiki.

**Reply by zhaolinlau** [reply]
> Perhaps you could put a link to it on the wiki.

Okay, I will put later
